### DISCUSSION TITLE
/dev/sda1 is not a valid choice

### DISCUSSION THREAD

**Discussion by COIh0rp** [retronas/discussions#60] [Q&A]
So i have installed retronas on a virtual machine running Debian 11 off my Synology NAS. I allocated 2 tb of storage to the virtual machine as the primary storage location. I am trying to set up retronas using cockpit. When I go to set the top level directory and use the 2 tb storage drive of the VM (/dev/sda1) it gives me the error that it is not a valid choice?

Is there any way to fix this? Is it because the storage drive is the primary drive for the VM and not an external drive?  The 2 tb drive is an ext 4 file system, but I cannot format it in cockpit as it is the primary drive for the VM.

Appreciate any help on this. I would like to be able to use the primary drive for the VM as the primary storage location for retronas

**Comment by sairuk** [comment]
/dev/sda1 is the raw device, you'll need to mount it into the filesystem to use it.

see the replies here https://www.reddit.com/r/retroNAS/comments/vcgnmy/devsda1_is_not_a_valid_choice/ as advised refer to https://www.youtube.com/watch?v=IyY80z5Vl_U for a demonstration on mounting devices with cockpit

once it is mounted, you will point RetroNAS at the mount point e.g. `/storage/retronas/`
